src/tbb/waiters.h

```diff
@@ -68,6 +68,7 @@ class outermost_worker_waiter : public waiter_base {
                     t2 = std::chrono::steady_clock::now())
                 {
                     if (!my_arena.is_empty() && !my_arena.is_recall_requested()) {
+                        reset_wait();
```

Comment:
I think such changes require performance and power consumption measurements at the very least. If this is done only for stabilizing the unit test results, and we do not have proofs the changes are acceptable I would not rush applying them.

Reply:
Ok, let's split the patch then? I suppose the test related changes are enough to stabilize the results.

Reply:
Sure.